src/test-utils/index.tsx

```diff
@@ -2,7 +2,7 @@ import {
     openPage,
     serverHostName,
     screen,
-    getGlobalPage,
+    globalPage,
```

Comment:
hmmm... I'd say I removed the `globalPage` export from the lib

Reply:
It exists in the d.ts file

Reply:
ok, I had an old library version